<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QArrangementLabellingInfoWidget</class>
 <widget class="QWidget" name="QArrangementLabellingInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>190</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>194</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Arrangement info</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/dialog-information.png</normaloff>:/dialog-information.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="_labelFaces">
     <property name="text">
      <string>0 faces</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_labelEdges">
     <property name="text">
      <string>0 edges</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_labelVertices">
     <property name="text">
      <string>0 vertices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_labelIsolatedVertices">
     <property name="text">
      <string>0 isolated vertices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_labelVerticesAtInfinity">
     <property name="text">
      <string>0 vertices at infinity</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_labelChanged">
     <property name="text">
      <string>No new changes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_labelCurrentFaceLabel">
     <property name="text">
      <string>current face label</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_labelNbArrangementsInHistory">
     <property name="text">
      <string>Number of arrangements in history</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
